ext/background/background.js

```diff
@@ -235,7 +235,13 @@ async function backfillHistory(session, provider) {
           return moment(b.time) - moment(a.time);
         });
         for (const t of tabs) {
-          await attemptRecordInteraction(t, session, false, t.time).then(() => count++);
+          try {
+            await attemptRecordInteraction(t, session, false, t.time);
+            count++;
+          } catch (e) {
+            console.log(e);
```

Comment:
```suggestion
            console.error(e);
```

Reply:
So, I don't want to log this as an error because it is fairly expected for this call to fail due to local and remote validation issue. It's fairly strict.